### PR TITLE
[CDAP-14121] Fixes 404 and 500 pages to look consistent.

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-14121
Build: https://builds.cask.co/browse/CDAP-UDUT92

### CODE REVIEW

File: cdap-ui/app/cdap/components/404/index.js [hunk 1]
```diff
@@ -27,7 +27,9 @@ export default function Page404({entityName, entityType, children}) {
   let namespace = NamespaceStore.getState().selectedNamespace;
   return (
     <div className="page-not-found">
-      <img src="/cdap_assets/img/404.png" />
+      <h1 className="error-main-title">
+        Page not found
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/404/index.js]: nit: i18n?

File: cdap-ui/app/cdap/components/500/index.js [hunk 2]
```diff
@@ -19,21 +19,37 @@ import React from 'react';
...
+          <strong>
+            {T.translate('features.Page404.subtitleMessage1')}
+          </strong>
+        </h4>
+        <div className="navigation-section">
+          <div>
+            {
+              // There is definitely a better way to do this :sigh:
+            }
+            {T.translate(`${I18N_PREFIX}.suggestion1Part1`)} <a href={window.location.href}> {T.translate(`${I18N_PREFIX}.suggestion1Part2`)} </a> {T.translate(`${I18N_PREFIX}.suggestion1Part3`)}
```
Comment (tonybach) [inline cdap-ui/app/cdap/components/500/index.js]: You can break this into 3 lines right, to make it more readable?

**Review by tonybach — COMMENTED** [review]
Minor comments

**Review by elfenheart — DISMISSED** [review]
👍 LGTM

**Review by bdmogal — APPROVED** [review]

**Pull request merged**